### PR TITLE
Set Vibe.d LogLevel along with our own

### PR DESCRIPTION
**Pull request by Geod24**
The log we get currently from Vibe are still pretty terrible.
See https://github.com/vibe-d/vibe-core/pull/212 for a first step at improving it.

### CODE REVIEW

File: source/agora/node/Runner.d [hunk 1]
```diff
@@ -73,3 +76,46 @@ public FullNode runNode (Config config)
     listenHTTP(settings, router);
     return node;
 }
+
+/*******************************************************************************
+
+    Set Vibe.d log level according to the configuration's log level
+
+    This is used to ensure we get the right amount og information from Vibe.d
```
Comment (AndrejMitrovic) [inline source/agora/node/Runner.d]:
```suggestion
    This is used to ensure we get the right amount of information from Vibe.d
```

File: source/agora/node/Runner.d [hunk 2]
```diff
@@ -73,3 +76,46 @@ public FullNode runNode (Config config)
...
+    Since a log level is the "minimum accepted" level, some level values might
+    not match. For example, if we want Vibe.d's "diagnostic" to be part of
+    the output when we set the loglevel to info, then that's what we must pass
+    to Vibe's `setLogLevel` (and since diagnostic < info, it will include the
+    later too).
```
Comment (AndrejMitrovic) [inline source/agora/node/Runner.d]:
```suggestion
    latter too).
```

File: tests/system/node/0/config.yaml [hunk 3]
```diff
@@ -54,4 +54,4 @@ network:
 ################################################################################
 logging:
   # Values: Trace, Info, Warn, Error, Fatal, None (default)
-  level: Trace
+  level: Info
```
Comment (AndrejMitrovic) [inline tests/system/node/0/config.yaml:57]: But why the change from trace to info in these tests? Is there too much vibe.d spam?

**Review by AndrejMitrovic — CHANGES_REQUESTED** [review]

File: tests/system/node/0/config.yaml (hunk 3 above)
Comment (Geod24) [reply tests/system/node/0/config.yaml:57]: >  Is there too much vibe.d spam?

It prints like 8 lines for EVERY request...

Comment (AndrejMitrovic) [reply tests/system/node/0/config.yaml:57]: Woops!

**Comment by Geod24** [comment]
Fiexd tpyos!

**Review by AndrejMitrovic — APPROVED** [review]
GTML!

File: source/agora/node/Runner.d [hunk 4]
```diff
@@ -73,3 +76,46 @@ public FullNode runNode (Config config)
...
+{
+    import vibe.core.log;
+
+    final switch (level)
+    {
+    case ILogger.Level.Trace:
+        setLogLevel(LogLevel.trace);
+        break;
+    case ILogger.Level.Info:
+        setLogLevel(LogLevel.diagnostic);
```
Comment (TrustHenry) [inline source/agora/node/Runner.d:106]: Why?  Isn't that `LogLevel.info`?

Comment (Geod24) [reply source/agora/node/Runner.d:106]: Look at the comment on the function

Comment (TrustHenry) [reply source/agora/node/Runner.d:106]: Vive.d LogLevel
diagnostic, /// Extended user information (e.g. for more detailed error information)
info,       /// Informational message for normal user education
I see.

**Review by TrustHenry — APPROVED** [review]
LGTM

**Review by linked0 — APPROVED** [review]
